source/changelog.txt

```diff
@@ -17,6 +17,24 @@ v2.0.0
 
 *Upcoming*
 
+- Added support for these constructors:
+
+  - :method:`Binary.createFromBase64()`
+  - :method:`ObjectId.createFromHexString()`
+  - ``Binary.createFromHexString()``
+  - ``ObjectId.createFromBase64()``
+
+- ``mongosh`` returns binary values as ``Binary.createFromBase64(
+  <base64String> )`` values instead of ``Binary( Buffer.from(
+  <base64String> ) )`` values. For example:
+
+  - Starting in 2.0.0: ``binaryValue:
+    Binary.createFromBase64("SGVsbG8gV29ybGQhCg==")``
+  - Earlier versions: ``binaryValue:
+    Binary(Buffer.from("SGVsbG8gV29ybGQhCg==", "hex"), 0)``
```

Comment:
Same comment as https://github.com/10gen/docs-mongodb-internal/pull/4411#discussion_r1304236034  applies here as well – this doesn’t look like current/1.x mongosh output.